Covid19Radar/Covid19Radar/ViewModels/Tutorial/TutorialPage3ViewModel.cs

```diff
@@ -44,12 +45,36 @@ IUserDataRepository userDataRepository
 
             UserDialogs.Instance.ShowLoading(Resources.AppResources.LoadingTextRegistering);
 
-            var registerResult = await _userDataService.RegisterUserAsync();
-            if (!registerResult)
+            try
+            {
+                var resultStatusCode = await _userDataService.RegisterUserAsync();
+                if (resultStatusCode != HttpStatusCode.OK)
+                {
+                    UserDialogs.Instance.HideLoading();
+                    if (resultStatusCode == HttpStatusCode.Forbidden)
+                    {
+                        _loggerService.Error("Failed register for requests from overseas");
+                        await UserDialogs.Instance.AlertAsync(
+                            Resources.AppResources.DialogNetworkConnectionErrorFromOverseasMessage,
+                            Resources.AppResources.DialogNetworkConnectionErrorTitle,
+                            Resources.AppResources.ButtonOk);
+                        _loggerService.EndMethod();
+                        return;
+                    }
+
+                    _loggerService.Error("Failed register");
+                    await UserDialogs.Instance.AlertAsync(Resources.AppResources.DialogNetworkConnectionError, Resources.AppResources.DialogNetworkConnectionErrorTitle, Resources.AppResources.ButtonOk);
```

Comment:
[修正任意]
catch 側で整形しているので、こちらも併せたほうが読みやすいです。

```suggestion
                    await UserDialogs.Instance.AlertAsync(
                        Resources.AppResources.DialogNetworkConnectionError,
                        Resources.AppResources.DialogNetworkConnectionErrorTitle,
                        Resources.AppResources.ButtonOk);
```

Reply:
修正しました。